docs_src/api/supporting/Ch-APISupportingServicesScheduler.md

```diff
@@ -0,0 +1,13 @@
+# Scheduler
```

Comment:
I have a PR for revamping all the API Reference, which this API update conflicts with. If you agree with my changes, please revert these changes so we don't have a merge conflict.
https://github.com/edgexfoundry/edgex-docs/pull/448

Reply:
Revert.